src/styles/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $card-border;
+  border-radius: 5px;
+  transition: color, transform $transition-duration $transition-property;
+
+  &:hover.card__title {
+    color: $color-card-title-hover;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    overflow: hidden;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-black;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-gray;
+  }
+
+  &__rating {
+    display: flex;
+    margin-top: 16px;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-black;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    height: 18px;
+    margin-top: 24px;
+
+  }
+
+  &__price {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $color-gray;
+  }
+
+  &__amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $color-black;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: block;
+    width: 166px;
+    height: 40px;
+    text-align: center;
+    line-height: 38px;
+    margin-inline: auto;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-white;
+    border-radius: 5px;
+    background-color: $color-blue;
+    border: 1px solid transparent;
+    transition: color,
+      background-color $transition-duration $transition-property;
```

Comment:
Make sure that you add all properties correctly and all hover effects work smoothly. Now it's true only for background-color
https://dev.to/clairecodes/css-transitions-for-multiple-properties-on-one-element-9o